app/mailers/user_notifier.rb

```diff
@@ -0,0 +1,12 @@
+class UserNotifier < ActionMailer::Base
+  default :from => 'any_from_address@example.com'
+
+  # Send notification email about a new issue
+  def send_notification_email(email, issue, label)
+    @email = email
+    @issue = issue
+    @label = label
+    mail( :to => @email,
+          :subject => "New issue (#{issue}) labeled #{label} created" )
```

Comment:
This subject seems very specific.  We will probably want to add other notification e-mails in the future, should this be in a subclass or something?